fdbcli/DebugCommands.actor.cpp

```diff
@@ -252,18 +253,20 @@ bool checkResults(Version version,
 		}
 	}
 
-	if (firstValidServer >= 0 && replies[firstValidServer].get().get().more) {
+	bool hasMore = firstValidServer >= 0 && replies[firstValidServer].get().get().more;
+	if (hasMore) {
 		const VectorRef<KeyValueRef>& result = replies[firstValidServer].get().get().data;
 		printf("Warning: Consistency check was incomplete, last key of server %d that was checked: %s\n",
 		       firstValidServer,
 		       printable(result[result.size() - 1].key).c_str());
-		begin = firstGreaterThan(result[result.size() - 1].key);
+		lastEnd = result[result.size() - 1].key; // store to a standalone, otherwise memory will be invalid
```

Comment:
@jzhou77 this is needed, because `result` is from `replies`, and `replies` will be clear at the begin of each round, so I fix it this way, please let me know if you have a better idea